schemas/config/2.0.schema.json

```diff
@@ -461,6 +461,73 @@
                 }
             }
         },
+        "otel": {
+            "description": "OpenTelemetry configuration",
+            "type": "object",
+            "properties": {
+                "insecure": {
+                    "description": "Use insecure endpoint",
+                    "type": "boolean",
+                    "default": false
+                },
+                "compress": {
+                    "description": "Use gzip compressor",
+                    "type": "boolean",
+                    "default": false
+                },
+                "exporter": {
+                    "description": "Provides functionality to emit telemetry to consumers",
+                    "type": "string",
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "zipkin",
+                            "stdout",
+                            "otlp",
+                            "jaeger",
+                            "jaeger_agent"
+                        ],
+                        "pattern": "^[0-9a-zA-Z_]+$"
+                    }
+                },
+                "custom_url": {
+                    "description": "Used for the http client to override the default URL",
+                    "type": "string",
+                    "default": ""
+                },
+                "endpoint": {
+                    "description": "Consumer's endpoint",
+                    "type": "string",
+                    "default": "localhost:4318"
```

Comment:
127.0.0.1, while `localhost` is also valid, it can be bonded to a different address.